v2/googlecloud-to-elasticsearch/src/main/resources/csv-to-elasticsearch-command-spec.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mainClass": "com.google.cloud.teleport.v2.elasticsearch.templates.CsvToElasticsearch",
+  "classPath": "/template/googlecloud-to-elasticsearch/*:/template/googlecloud-to-elasticsearch/libs/*:/template/googlecloud-to-elasticsearch/classes",
```

Comment:
Mainclass can be renamed to com.google.cloud.teleport.v2.elasticsearch.templates.GCSToElasticsearch
classpath should be gcs-to-elasticsearch instead of googlecloud-to-elasticsearch
e.g: /template/gcs-to-elasticsearch/